src/CSF.SRDashboard.Client/Components/RequestStatus.razor.cs

```diff
@@ -39,9 +39,17 @@ public EditContext EditContext
         [Inject]
         IStringLocalizer<Shared.Common> Localizer { get; set; }
 
+        [Inject]
+        IStringLocalizer<RequestStatus> Localizer2 { get; set; }
```

Comment:
I suggest renaming Localizer2 to something a bit more explicit.
```suggestion
        IStringLocalizer<RequestStatus> RequestStatusLocalizer { get; set; }
```